src/styles/blocks/testimonials.scss

```diff
@@ -0,0 +1,41 @@
+.testimonials {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__title {
+    text-align: left;
+    width: max-content;
+    margin-bottom: 32px;
+
+    @include onTablet {
+      font-size: 40px;
+      line-height: 40px;
+      margin-right: 10rem;
+
+    }
+
```

Comment:
there is something wrong with the indents

![Screenshot_7](https://user-images.githubusercontent.com/89162624/208910227-ca1df4d6-004d-42f6-b2e7-b89651b6a9e7.png)